### PR TITLE
Init codeowners

### PR DESCRIPTION
**Pull request by mojito317**
## PR summary

The reason for this PR to restrict pull requests against files that are generated.

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [x] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [x] Build/CI related changes
- [x] Documentation content changes
- [ ] Other (please describe)

## What is the current behavior?
PRs can be merged for generated files without the approval of sdks team members.

## What is the new behavior?

The listed files in [CODEOWNERS](.github/CODEOWNERS) can no longer be pushed before the @IBM/cloudant-sdks members have not reviewed it.
This also restricts the PRs from @cloudant-sdks-automation: the @IBM/cloudant-sdks must review this PR before the merge.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+# Generated code files:
+
+cloudant/v1.ts @IBM/cloudant-sdks
+test/unit/cloudant.v1.test.js @IBM/cloudant-sdks
+
+# Bump version files:
+
+.bumpversion.cfg @IBM/cloudant-sdks
+# README.md
```
Comment (emlaver) [inline .github/CODEOWNERS:18]: I suggest we remove the commented lines.  Same suggestion in https://github.com/IBM/cloudant-java-sdk/pull/47#discussion_r528755420

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# package-lock.json
+# package.json
```
Comment (emlaver) [inline .github/CODEOWNERS:20]: File needs new line

**Review by emlaver — APPROVED** [review]

**Review by bessbd — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (mojito317) [reply .github/CODEOWNERS]: As we agreed with @ricellis offline. The package files will stay uncommented, README does not, till it is not generated too.

File: .github/CODEOWNERS (hunk 2 above)
Comment (mojito317) [reply .github/CODEOWNERS]: Fix in dbce6be.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# Generated code files:
+
+cloudant/v1.ts @IBM/cloudant-sdks
+test/unit/cloudant.v1.test.js @IBM/cloudant-sdks
+
+# Bump version files except the README:
+
+.bumpversion.cfg @IBM/cloudant-sdks
+package-lock.json
+package.json
```
Comment (ricellis) [inline .github/CODEOWNERS:19]:
```suggestion
package-lock.json @IBM/cloudant-sdks
package.json @IBM/cloudant-sdks
```

Comment (mojito317) [reply .github/CODEOWNERS]: Thanks, rebased the file with the suggestion in c1e6c2e.

**Review by ricellis — APPROVED** [review]

**Comment by mojito317** [comment]
Thanks for the reviews @emlaver, @bessbd, and @ricellis! I'm about to merge this.

**Pull request merged**